### PR TITLE
Added a video tutorial

### PR DESCRIPTION
**Pull request by smarthomejunkie**
## Proposed change
Added a video tutorial that explains how to set up the Google Sheets integration in Home Assistant and how you can add data from Home Assistant in a Google sheet.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.
- [X] Added a video tutorial

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/google_sheets.markdown [hunk 1]
```diff
@@ -72,6 +72,11 @@ The integration setup will next give you instructions to enter the [Application
 
 If you have an error with your credentials you can delete them in the [Application Credentials](/integrations/application_credentials/) user interface.
 
+### Video Tutorial
+This video tutorial explains how to set up the Google Sheets integration and how you can add data from Home Assistant in a Google Sheet.
```
Comment (c0ffeeca7) [inline source/_integrations/google_sheets.markdown]:
```suggestion
This video tutorial explains how to set up the Google Sheets integration and how you can add data from Home Assistant to a Google Sheet.
```

Comment (c0ffeeca7) [reply source/_integrations/google_sheets.markdown]: tiny tweak

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

**Review by c0ffeeca7 — APPROVED** [review]
Nice! Thank you, @smarthomejunkie :+1:

**Comment by smarthomejunkie** [comment]
Thank you for the merge! <3

**Pull request merged**